nifi-docs/src/main/asciidoc/record-path-guide.adoc

```diff
@@ -869,7 +869,42 @@ The following record path expression would append '@' characters to the input St
 | `padRight(/name, 15, '@')` | john smith@@@@@
 |==========================================================
 
+=== uuid5
```

Comment:
I find the doc a little confusing since the first case has example output and the second doesn't. Might also want to indicate the errors cases, which I think is just empty or invalid namespace strings and null names.

Reply:
I rewrote the documentation, so it should be good now.